src/migrations/tasks/09.add-last-updated-data.ts

```diff
@@ -0,0 +1,47 @@
+import { baseConfig } from "../../models/Config";
+import { D2Api } from "../../types/d2-api";
+import { addAttributeValueToObj } from "../../models/Attributes";
+import { Debug, Migration } from "../types";
+import { post } from "./common";
+
+class AddLastUpdatedDataMigration {
```

Comment:
The migration was not being executed. The trick is to search some existing migration name/file in the code to see how it's referenced. Then you get to `src/migrations/tasks/index.ts`.